cfme/configure/access_control.py

```diff
@@ -1237,6 +1238,10 @@ def is_displayed(self):
 class DetailsTenantView(ConfigurationView):
     """ Details Tenant View """
     toolbar = View.nested(AccessControlToolbar)
+    name = Text('Name')
+    description = Text('Description')
+    parent = Text('Parent')
+    table = Table('//*[@id="fieldset"]/table')
```

Comment:
**REQUIRED**
This should be replaced by [`BaseListEntity`](https://github.com/ManageIQ/integration_tests/blob/master/widgetastic_manageiq/__init__.py#L2414).